src/audio/AudioManager.cpp

```diff
@@ -238,3 +250,54 @@ void AudioManager::loadEffect(const char *fileName){
     chun[fileName] = sound;
 #endif
 }
+
+/**
+*   Function: fadeMusicOut(int ms)
+*               int ms: amount of milliseconds to fade out music
+*
+*   Returns: void
+*
+*   Date: April 5, 2017
+*
+*   Programmer: Alex Zielinski
+*
+*   Description:
+*       Wrapper fucntion to fade music out. The amount of time is takes to fade
+*       the music out is specified by the paramater int ms (in milliseconds)
+*/
+void AudioManager::fadeMusicOut(int ms){
+
+    Mix_FadeOutMusic(ms);
+}
+
+/**
+*   Function: playMenuMusic(const char *fileName1, const char *fileName2)
+*               const char *fileName1: first menu music audio file
+*               const char *fileName2: second menu music audio file
+*
+*   Returns: void
+*
+*   Date: April 5, 2017
+*
+*   Programmer: Alex Zielinski
+*
+*   Description:
+*       Randomly selects one of the 2 menu background music audio files
+*       to play when the menu loads
+*/
+void AudioManager::playMenuMusic(const char *fileName1, const char *fileName2){   // generates either 1 or 2
+
```

Comment:
Can you please wrap all audio manager functions with the ifndef server preprocessor directives?
The server should not be playing sound, so it's easiest to make the audio manager do nothing that to hunt down every sound effect call
Look at other calls for the format of the directives
Thanks Alex